haddock-api/src/Haddock/Interface.hs

```diff
@@ -165,6 +167,7 @@ createIfaces verbosity flags instIfaceMap mods = do
 processModule :: Verbosity -> ModSummary -> [Flag] -> IfaceMap -> InstIfaceMap -> Ghc (Maybe Interface)
 processModule verbosity modsum flags modMap instIfaceMap = do
   out verbosity verbose $ "Checking module " ++ moduleString (ms_mod modsum) ++ "..."
+  liftIO $ hSetEncoding stderr utf8
```

Comment:
Sorry, mistaken.
I failed to build cabal-install with GHC 8.2 on my Windows. Actually, I successfully installed GHC 8.2.